### PR TITLE
1010 data deletion details not hidden

### LINKED ISSUE #1010: Data deletion details not hidden in capability view page
### Description
In catalog, the data deletion endpoint details "Api Key" field should be hidden and be able to toggle on eye-icon. Scrrenshot below shows the current capability view page (no eye-icon in data deletion endpoint).
Screen Shot 2022-09-15 at 9 40 13 AM

### Acceptance Criteria
Data deletion endpoint "Api Key" field is hidden and is able to toggle on eye-icon click.

### PR DESCRIPTION
**Pull request by minump**
## Description
Toggle eye-icon added to data deletion endpoint details in capability view page.

**Fixes #1010**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by sandeep-ps** [comment]
Other changes look good to me. Thanks.

File: contributions/catalog/webapps/templates/contribute/capability_details.html
Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/capability_details.html]: Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
Approved. 
Hi, @minump, Please merge changes from develop. Please check with @bingzhang for any feedback. You can merge this PR after his approval. Thanks.

**Review by bingzhang — APPROVED** [review]

**Review by sandeep-ps — APPROVED** [review]

**Comment by minump** [comment]
@sandeep-ps @bingzhang Can you please approve this once more. Need an approval before merge.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**
